### ISSUE TITLE
Connection timed out

### ISSUE THREAD

**Issue by eternalsayed** [defunctzombie/localtunnel-server#113]
I've been able to setup localtunnel-server on my digitalOcean droplet successfully running LAMP stack. I've setup the app behind apache proxy (not a dev ops person; just know the basics of setting up a server). I could also get url for my client when I run
`lt -h sub.domain.com -p 9292` for server running on my client. However, when I open the URL in my browser, it keeps loading and then gives connection timed out.
My lt-server runs on port 6000 behind proxied port 80.
Wondering if someone can help me understand the problem?
I could get status successfully as below, if I run `sub.domain.com/api/status`:
`{"tunnels":0,"mem":{"rss":32759808,"heapTotal":31981568,"heapUsed":16313712,"external":236228}}`

**Comment by bmkimagem** [comment]
You solved it?? I have the same problem

**Comment by bmkimagem** [comment]
@eternalsayed You solved it?? I have the same problem

**Comment by eternalsayed** [comment]
No. The problem was never fixed. I tried to setup the server on my mac and it worked fine. Probably something to do with my VM config. I'll try it on my Raspberry pi and see if it works. Will update when I'll do that.
